File: 04_Access_Modifiers_and_Encapsulation/Box.cpp
#include <iostream>
using namespace std;

class Box {
private:
    int length;
    int width;

public:
    void displayBox() {
        cout << "Box dimensions: " << this->length << " x " << this->width << endl;
    }
};

int main() {
    Box box;
    box.displayBox();
    // cout << box.length << endl; // member "Box::length" (declared at line 6) is inaccessible
    return 0;
}

File: 04_Access_Modifiers_and_Encapsulation/Box.java
class Box {
    private int length;
    private int width;

    public void displayBox() {
        System.out.println("Box dimensions: " + length + " x " + width);
    }

    public static void main(String[] args) {
        Box box = new Box();
        box.length = 10;
        box.width = 20;

        box.displayBox();
    }
}
